wiki/entry_schmitz.qmd

```diff
@@ -0,0 +1,45 @@
+--
+title: "Wiki Entry Template"
+subtitle: "BA New Media / Online Communication KF/BF A: Spring Term 2023"
+author: Ronja Schmitz 
+categories:
+  - "Media Psychology"
+  - "Digital Media"
+  - "Well Being"
+---
+
+Children and Adolescents and Digital Media 
+
+Key Quote: 
+
+“Thus, digital media can provide an engaging experience in which the media experiences of children and teenagers become highly personalized.“
+
+Argument
+ 
+Der Text befasst sich insbesondere mit den Risiken der (online) Mediennutzung für Kinder und Jugendliche. Es werden die verschiedenen Altersstufen betrachtet und welche individuellen Gefahren die Mediennutzung für die jeweilige Altersgruppe darstellen kann. Es wird an dieser Stelle betont, dass man davon ausgehen muss, dass ebenfalls Persönlichkeit und die sozialen Umstände eine wichtige Rolle dabei spielen, inwiefern die Nutzung zur Gefahr wird. 
```

Comment:
You mention that social circumstances have an inpact on the risks if online media use. So I ask myself, which social circumstances are an inpact? Because there are several social factors like education, occupation, income, and integration into sociaty and so an. If it is mention in the paper, it would be awesome if you could give some examples or name some of the social circumstances.